vendor/vendor.json

```diff
@@ -1,660 +1,660 @@
 {
-	"comment": "",
```

Comment:
can you keep the original indent so the diff is more visible?

Reply:
👍